docs/topics/multiplatform/multiplatform-compatibility-guide.md

```diff
@@ -490,3 +490,67 @@ projects.
 **When do the changes take effect?**
 
 In Kotlin 1.9.0, a deprecation warning is introduced when the `android` name is used in Kotlin Multiplatform projects.
+
+<anchor name="declaring-multiple-targets"></anchor>
+**What's changed?**
+
+Declaring multiple targets of the same kind in one project is not recommended and will be forbidden in future releases.
+There are multiple ways of achieving same results with other Kotlin Gradle Plugin mechanisms. As part of Kotlin
```

Comment:
I'd drop those two sentences, we're expanding on this in details below anyways

Reply:
But I think something should be placed under "what's changed" section.
I'll try to rephrase it to something like:

> As part of Kotlin Multiplatform stabilization we are trying to emphasize the essential and reduce ambiguous APIs. Therefore we discourage declaring multiple targets of the same kind in a single Gradle project.

The first sentence can also be dropped as it is just a "soft" introduction to what exactly we changed. But it can blur the reader's eye.

Reply:
> The first sentence can also be dropped

Yes, I'd agree

Reply:
Updated